bridge/bridge.go

```diff
@@ -225,6 +228,10 @@ func (b *Bridge) SetIRCListenerName(name string) {
 	b.ircListener.Nick(name)
 }
 
+func (b *Bridge) SetIRCPassword(password string) {
```

Comment:
I don't think this method is necessary, as the config given is passed around the bridge anyway. i.e. server pass should still work if you remove this function (and the call to this function)